homeassistant/components/switchmate/switch.py

```diff
@@ -17,7 +17,7 @@
 CONF_FLIP_ON_OFF = "flip_on_off"
 DEFAULT_NAME = "Switchmate"
 
-SCAN_INTERVAL = timedelta(minutes=30)
+SCAN_INTERVAL = timedelta(seconds=10)
```

Comment:
Thats a pretty significant scan interval change for a device which polls.  Do these devices advertise their state, if so we can listen for advertisements with https://developers.home-assistant.io/docs/network_discovery?_highlight=discovery#subscribing-to-bluetooth-discoveries


Also please see https://developers.home-assistant.io/docs/network_discovery?_highlight=discovery#best-practices-for-library-authors

Reply:
Check the switchbot integration for an example of push updates